ads/google/a4a/test/test-traffic-experiments.js

```diff
@@ -239,11 +230,8 @@ describe('all-traffic-experiments-tests', () => {
           experiment: '108642',
         },
       };
-      sandbox.win.AMP_CONFIG = {};
-      const config = sandbox.win.AMP_CONFIG;
-      config['fooExpt'] = 0.0;
+      toggleExperiment(sandbox.win, 'fooExpt', false, true);
       randomlySelectUnsetPageExperiments(sandbox.win, exptAInfo);
-      config['fooExpt'] = 1.0;
```

Comment:
I will remove this test case if no objection. Since the goal of this PR is to guarantee an immutable AMP_CONFIG (no one should change it), there's no point to have such a test.
@tdrl